geometry_msgs/Point[] robot_pos
geometry_msgs/Point[] box_pos
